### PR TITLE
Update BSR badge to beta

### PR DESCRIPTION
**Pull request by lucperkins**

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ const sidebars = {
           label: "Remote Generation",
           customProps: {
             badge: {
-              label: "alpha",
+              label: "beta",
```
[the remark this answers is not in the document]
Comment (lucperkins) [reply sidebars.js:82]: Fair question. I'll defer to @SyrieBianco on this.

[the remark this answers is not in the document]
Comment (SyrieBianco) [reply sidebars.js:82]: I think I'd say it's beta — it's publicly available. I think it was beta when we first opened it to public use, we were just trying to emphasize that it's under development.

@johanbrandhorst we don't think that the URLs and synthetic versioning conventions are likely to change, do we? If we think they're 85% stable, I'd say we *don't* need to de-emphasize this section as alpha.

[the remark this answers is not in the document]
Comment (johanbrandhorst) [reply sidebars.js:82]: Agreed, sounds good to me

**Review by SyrieBianco — APPROVED** [review]
Pending an objection from @johanbrandhorst or @bufdev, I'd say this is a appropriate

File: sidebars.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (bufdev) [reply sidebars.js:82]: I think it's still alpha - we have plans to redo this a bit

**Pull request merged**